bigbluebutton-html5/public/locales/en.json

```diff
@@ -624,6 +628,8 @@
     "app.settings.dataSavingTab.webcam": "Enable other participants webcams",
     "app.settings.dataSavingTab.screenShare": "Enable other participants desktop sharing",
     "app.settings.dataSavingTab.description": "To save your bandwidth adjust what's currently being displayed.",
+    "app.settings.transcriptionTab.partialUtterances": "Show partial utterances",
+    "app.settings.transcriptionTab.minUtteranceLength": "Minimum utterances length (seconds)",
```

Comment:
Have you considered renaming the labels to be more descriptive? For example: "Length of the speech (in seconds) used for transcription. Longer portion typically transcribe better but also result in longer delay before displaying"